libvisca.lua

```diff
@@ -582,7 +582,7 @@ function Visca.ReplyServer.add_listener_for(address, port)
     if cnt == 0 then
         local sock_address, err, num = socket.find_first_address("*", port,
             {family="inet", socket_type="dgram", protocol="udp"})
-        if address then
+        if sock_address then
```

Comment:
I assume you meant sock_address here since you want to print err on the failure case